proto/crosschain/tx.proto

```diff
@@ -75,7 +75,9 @@ message MsgAddToOutTxTracker {
   int64 tx_index = 7;
 }
 
-message MsgAddToOutTxTrackerResponse {}
+message MsgAddToOutTxTrackerResponse {
+  bool is_removed = 1; // if the tx was removed from the tracker due to no pending cctx
```

Comment:
It seems it is not used in the message handler returns

Reply:
most cases defaults to false which is correct; 
there is one place it is set to true. 
Can add more explicit assignment but maybe not littering is better as the is_remove=true is really special case.

Reply:
I agree to not specify the value if it is false.
But it seems it is not used when it should be true: https://github.com/zeta-chain/node/pull/1370/files#diff-9ef5905d374779457a649b7c98c6b5ed348612bcb1a43a30d0923fecb27e86bfR64

Reply:
ah, yes
fixed here https://github.com/zeta-chain/node/pull/1370/commits/9edf3e9afc6772a3d747fc4ca7c1235286fb46c3